packages/dds/sequence/src/sharedStringWithSequentialId.ts

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IMergeTreeOp, Marker,
+    MergeTreeDeltaType, reservedMarkerIdKey } from "@fluidframework/merge-tree";
+import { ISequencedDocumentMessage, MessageType } from "@fluidframework/protocol-definitions";
+import { parseHandles } from "@fluidframework/shared-object-base";
+import { createIdAfterMin, createIdBeforeMax } from "./generateSequentialId";
+import { SharedString } from "./sharedString";
+
+export function sharedStringWithSequentialIdMixin(Base: typeof SharedString = SharedString): typeof SharedString {
+    return class SequenceWithSequentialId extends Base {
+        protected processMergeTreeMsg(
+            rawMessage: ISequencedDocumentMessage): void {
+                const message = parseHandles(
+                    rawMessage,
+                    this.runtime.IFluidSerializer);
+                this.applySequentialId(message);
+                super.processMergeTreeMsg(rawMessage);
```

Comment:
A bit confused on usage of message vs. rawMessage. Isn't it a bug that we pass rawMessage here?
It does not feel it would work, given that parseHandles created a deep copy of the message.
Do tests pass with that?

Reply:
No longer using the raw message after not doing processTreeMsg